docs/source/tutorial/bft_raft_migration_tutorial.md

```diff
@@ -0,0 +1,162 @@
+
+```bash
+peer channel fetch config config_block.pb -o orderer.example.com:7050 -c $CHANNEL_NAME --tls --cafile $ORDERER_CA
+
+configtxlator proto_decode --input config_block.pb --type common.Block | jq .data.data[0].payload.data.config > current_config.json
+
+# Check current consensus type
+jq '.channel_group.groups.Orderer.values.ConsensusType.value.type' current_config.json
+
+# Check current consenter mapping
+jq '.channel_group.groups.Orderer.values.Orderers.value.consenter_mapping' current_config.json
+```
+
+## Step 3: Enter Maintenance Mode
+
+```bash
+cp current_config.json modified_config.json
+
+jq '.channel_group.groups.Orderer.values.ConsensusType.value.state = "STATE_MAINTENANCE"' modified_config.json > updated_config.json
+
+configtxlator proto_encode --input current_config.json --type common.Config --output current_config.pb
+configtxlator proto_encode --input updated_config.json --type common.Config --output updated_config.pb
+configtxlator compute_update --channel_id $CHANNEL_NAME --original current_config.pb --updated updated_config.pb --output config_update.pb
+
+peer channel signconfigtx -f config_update.pb
+peer channel update -f config_update.pb -c $CHANNEL_NAME -o orderer.example.com:7050 --tls --cafile $ORDERER_CA
+```
+
+## Step 4: Backup the System
+
+```bash
+docker stop $(docker ps -q --filter name=orderer)
+tar -czf orderer_backup.tar.gz /var/hyperledger/production/orderer
+docker start $(docker ps -aq --filter name=orderer)
+```
+
+## Step 5: Switch to BFT and Update Consenter Mapping
+
+```bash
+# Prepare the new consenter mapping
+cat << EOF > consenter_mapping.json
+[
+  {
+    "id": 1,
+    "host": "orderer1.example.com",
+    "port": 7050,
+    "msp_id": "OrdererMSP",
+    "identity": "base64_encoded_identity",
+    "client_tls_cert": "base64_encoded_client_cert",
+    "server_tls_cert": "base64_encoded_server_cert"
+  },
+  {
+    "id": 2,
+    "host": "orderer2.example.com",
+    "port": 7050,
+    "msp_id": "OrdererMSP",
+    "identity": "base64_encoded_identity",
+    "client_tls_cert": "base64_encoded_client_cert",
+    "server_tls_cert": "base64_encoded_server_cert"
+  },
+  {
+    "id": 3,
+    "host": "orderer3.example.com",
+    "port": 7050,
+    "msp_id": "OrdererMSP",
+    "identity": "base64_encoded_identity",
+    "client_tls_cert": "base64_encoded_client_cert",
+    "server_tls_cert": "base64_encoded_server_cert"
+  },
+  {
+    "id": 4,
+    "host": "orderer4.example.com",
+    "port": 7050,
+    "msp_id": "OrdererMSP",
+    "identity": "base64_encoded_identity",
+    "client_tls_cert": "base64_encoded_client_cert",
+    "server_tls_cert": "base64_encoded_server_cert"
+  }
+]
+EOF
+
+# Update the config with new consensus type, metadata, and consenter mapping
+jq --slurpfile mapping consenter_mapping.json '
+  .channel_group.groups.Orderer.values.ConsensusType.value.type = "BFT" |
+  .channel_group.groups.Orderer.values.ConsensusType.value.metadata = {
```

Comment:
these are Raft metadata parameters, why would we use them?

In BFT we have the following: https://github.com/hyperledger/fabric/blob/main/sampleconfig/configtx.yaml#L686-L697

Reply:
Thanks @yacovm for the catch added the correct meta parameters.